packages/ui-extensions/src/surfaces/checkout/api/address-autocomplete/address-autocomplete.ts

```diff
@@ -14,23 +26,18 @@ export interface AddressAutocompleteSuggestApi {
      *
      * @example "address1"
      */
-    field: AutocompleteAddress;
-
-    /**
-     * An AbortSignal that may be used to cancel in-flight network requests.
-     * This signal is sent when the buyer has stopped interacting with the address field.
-     */
-    signal: AbortSignal;
+    field: Extract<keyof MailingAddress, 'address1' | 'zip'>;
```

Comment:
This type is changing in Haylee's doc PR. So you'll need to do another unstable release to bring that change in. No problem with that approach though.